README.md

```diff
@@ -1 +1,21 @@
-# data-monitoring-dashboard
\ No newline at end of file
+# data-monitoring-dashboard
+
+## Purpose
+
+## Technical Architecture Diagram
+Coming soon
+
+## How to set up dev environment and run tests
+Coming soon
+
+## How to setup and deploy to a new environment
+Coming soon
+
+## How to deploy api and frontend to existing environment
+Coming soon
+
+## How to build and deploy data refresher docker image to existing environment
+Coming soon
+
+## How to get test data for development
+Coming soon
```

Comment:
It would be clearer if those `How-tos` were under a parent section called "Development" or "Deployment", which is living alongside the "Purpose" and "Technical Architecture Diagram"

Reply:
This makes sense! I'll structure it this way in the PR that has the actual read-me content